vnext/Microsoft.ReactNative/ReactApplication.cpp

```diff
@@ -96,7 +96,10 @@ void ReactApplication::JavaScriptBundleFile(hstring const &value) noexcept {
 void ReactApplication::OnActivated(Windows::ApplicationModel::Activation::IActivatedEventArgs const &e) {
   if (e.Kind() == Windows::ApplicationModel::Activation::ActivationKind::Protocol) {
     auto protocolActivatedEventArgs{e.as<Windows::ApplicationModel::Activation::ProtocolActivatedEventArgs>()};
-    ::Microsoft::ReactNative::LinkingManagerModule::OpenUri(protocolActivatedEventArgs.Uri());
+    if (InstanceSettings().InitialUrl().empty()) {
```

Comment:
I considered implementing logic to prevent the InitialUrl from being set twice, but I think it's probably better to let developers decide what constitutes a change to the initial URL, and provide a similar default behavior in ReactApplication.